src/Microsoft.AspNet.Authentication/AuthenticateResult.cs

```diff
@@ -44,12 +44,17 @@ public static AuthenticateResult Success(AuthenticationTicket ticket)
             return new AuthenticateResult() { Ticket = ticket };
         }
 
-        public static AuthenticateResult Failed(Exception error)
+        public static AuthenticateResult Skip()
+        {
+            return new AuthenticateResult() { Skipped = true };
+        }
+
+        public static AuthenticateResult Fail(Exception error)
```

Comment:
Or maybe rename Fail to Error, with the property ErrorException is maybe better?